### ISSUE TITLE
Tray Icons disappear after upgrade to 0.11.0

### ISSUE THREAD

**Issue by pradyparanjpe** [Alexays/Waybar#3607]
Attached: screenshot with ```--log-level warning```

Here are some details:

OS: Arch Linux x86_64 
Kernel: 6.10.10-arch1-1
WM: sway 

```sh
$ paru -Qs ayatana
```
```
local/ayatana-ido 0.10.4-1
    
local/libappindicator-gtk3 12.10.0.r298-4
    Allow applications to extend a menu via Ayatana indicators in Unity, KDE or Systray (GTK+ 3 library)
local/libayatana-appindicator 0.5.93-1
    
local/libayatana-indicator 0.9.4-1
```

Duplicate of issue #3597.

![scrn-2024-09-15-16-56-04](https://github.com/user-attachments/assets/a169a320-62c9-4e1b-b075-1eab069a7a91)

**Comment by pradyparanjpe** [comment]
> I had the same problem, but could resolve it by removing the package waybar 0.11.0-2 and installing the waybar git version, at least that worked in my case.

That did work.

I'll have to watch waybar release for a fixed update though...

**Comment by itz-Amethyst** [comment]
Same here.

**Comment by yuheho7749** [comment]
It's fixed for me after upgrading to waybar-0.11.0-3

**Comment by pradyparanjpe** [comment]
Duplicate of issue #3597
